### DISCUSSION TITLE
RYB color wheel instead of RGB

### DISCUSSION THREAD

**Discussion by quentindecaillet** [jaames/iro.js#174] [Feature Requests]
Hi there :) I've tried to look through all the documentation, but it seems like the color wheel can only be displayed in using the RGB model, and not the RYB. Is there any plan to implement the choice between the two in the future?

**Comment by jaames** [comment]
That's an interesting one! At the moment there isn't a way to do this, and implementing it would add a bit of complexity to the colorwheel rendering and input handling for sure. It uses the HSV model so it's pretty easy to compute; saturation is just the distance from the centre, hue is the angle.

Can I ask what the use-case is here? I like to keep this library small so I usually only consider features if they're going to be widely useful :)

**Comment by quentindecaillet** [comment]
Thanks for the answer :) It's to integrate the wheel into Photoshop panels I create. I find the RYB model much more pleasing when it comes to creating color harmonies for color grading purposes or digital painting/drawing.
I've found code here and there on the web to make the conversion from RGB to RYB, but haven't looked in depth as to where I'd need to implement it in IRO to make the color wheel display work. I may look into it if that's not on the roadmap for now.
